apps/github/src/@types/index.d.ts

```diff
@@ -0,0 +1,30 @@
+type Payload = {
+    action: string;
+    sender: { [key: string]: unknown };
+    repository: { [key: string]: unknown };
+    organization: { [key: string]: unknown };
+    installation: { [key: string]: unknown };
+    [key: string]: unknown;
+};
+
+type Issue = {
+    body: string | null;
+    title: string;
+    [key: string]: unknown;
+};
+
+type Config = {
+    PRIVATE_REPO_NAME: string | undefined;
+    PUBLIC_REPO_NAME: string | undefined;
+    ORGANIZATION_NAME: string | undefined;
+    TOKEN: string | undefined;
```

Comment:
These can be optional typed.
```suggestion
    PRIVATE_REPO_NAME?: string;
    PUBLIC_REPO_NAME?: string;
    ORGANIZATION_NAME?: string;
    TOKEN?: string;
```

Reply:
Done.